src/graph/info.h

```diff
@@ -47,9 +47,9 @@ typedef struct QueryInfo {
 
 typedef struct FinishedQueryInfo {
     uint64_t received_unix_timestamp_milliseconds;
-    uint64_t total_waited_time_milliseconds;
-    uint64_t total_executed_time_milliseconds;
-    uint64_t total_reported_time_milliseconds;
+    uint64_t total_wait_duration_milliseconds;
```

Comment:
I think 32 bits would be enough for all durations (it limits queries to 49.7 days, which is... enough)